src/test/java/com/sonyericsson/rebuild/matchers/IsCollectionContainingStringParameterValues.java

```diff
@@ -0,0 +1,47 @@
+package com.sonyericsson.rebuild.matchers;
+
+import hudson.model.ParameterValue;
+import org.hamcrest.Description;
+import org.junit.internal.matchers.TypeSafeMatcher;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public final class IsCollectionContainingStringParameterValues extends TypeSafeMatcher<Collection<ParameterValue>> {
+
```

Comment:
I will refactor this to be the `StringParameterValuesMatcher` and I would delete the `IsNotCollectionContainingStringParameterValues ` and the current `StringParameterValuesMatcher` (from this class you just need the `matches` method since the constructor just takes the `StringParameterValue` constructor).